packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcStart.ts

```diff
@@ -147,6 +148,10 @@ export class ForceLightningLwcStartExecutor extends SfdxCommandletExecutor<{}> {
         if (data.toString().includes(errorHints.INACTIVE_SCRATCH_ORG)) {
           this.errorHint = errorHints.INACTIVE_SCRATCH_ORG;
         }
+        if (data.toString().includes('No \'lwc\' directory found')) {
+          errorCode = 135;
+          this.errorHint = errorHints.LWC_FOLDER_NOT_FOUND;
```

Comment:
I think you're okay to go with either the errorCode or the errorHint approach, both aren't needed though. The reason why errorHint was originally added was to further clarify the server startup reason if we had additional information from stderr (provided from the lwc dev server).